### PR TITLE
Updated description on follow camera script

### PR DESCRIPTION
**Pull request by yaustar**
Forgot to commit suggestion on #3696 

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: scripts/camera/follow-camera.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ FollowCamera.attributes.add('lerpAmount', {
     max: 1,
     default: 0.99,
     title: 'Lerp Amount',
-    description: 'The amount to lerp the camera towards its desired position every frame based on a 60fps frame rate. Lerping is frame rate independent though and will be correct for other frame rates.'
+    description: 'The amount to lerp the camera towards its desired position over time. Adjust the value until it feels right for the app. Lerping is frame rate independent though and will be correct for other frame rates.'
```
Comment (mvaligursky) [inline scripts/camera/follow-camera.js:20]: personally, the original text feels more like something you can work with and understand, than the new text?

Comment (yaustar) [reply scripts/camera/follow-camera.js:20]: The old text is no longer correct. The lerp amount is ultimately a 'fudge' number that is changed until it 'feels' right.

Comment (mvaligursky) [reply scripts/camera/follow-camera.js:20]: yeah I like that .. gives them some useful info

Comment (yaustar) [reply scripts/camera/follow-camera.js:20]:
```suggestion
    description: 'The amount to lerp the camera towards its desired position over time. The closer it is to 1, the faster the camera will move. Lerping is frame rate independent though and will be correct for other frame rates.'
```
```suggestion
    description: 'The amount to lerp the camera towards its desired position over time. Adjust the value until it feels right for the app. Lerping is frame rate independent though and will be correct for other frame rates.'
```

**Comment by yaustar** [comment]
Updated the description :)

File: scripts/camera/follow-camera.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ FollowCamera.attributes.add('lerpAmount', {
     max: 1,
     default: 0.99,
     title: 'Lerp Amount',
-    description: 'The amount to lerp the camera towards its desired position every frame based on a 60fps frame rate. Lerping is frame rate independent though and will be correct for other frame rates.'
+    description: 'The amount to lerp the camera towards its desired position over time. The closer it is to 1, the faster the camera will move. Lerping is frame rate independent though and will be correct for other frame rates.'
```
Comment (mvaligursky) [inline scripts/camera/follow-camera.js:20]:
```suggestion
    description: 'The amount to lerp the camera towards its desired position over time. The closer it is to 1, the faster the camera will move. Lerping is frame rate independent and will be correct for every frame rate.'
```

**Review by mvaligursky — APPROVED** [review]

**Pull request merged**